modules/microshift-lvm-thin-volumes.adoc

```diff
@@ -46,4 +46,9 @@ device-classes: <2>
 [IMPORTANT]
 ====
 When multiple PVCs are created simultaneously, a race condition prevents LVMS from accurately tracking the allocated space and preserving the storage capacity for a device class. Use separate volume groups and device classes to protect the storage of highly dynamic workloads from each other.
+====
```

Comment:
I have added "important" here as the content in the additional resources below this section lead to "Creating and managing thin-provisioned volumes (thin volumes)". The placement of this admonition can be modified. Please suggest about the placement of this admonition in microshift docs and about the the auto expansion in microshift

Reply:
The change seems fine to me

Reply:
what has to be added in microshift regarding the auto expansion?

Reply:
> anything has to be added in microshift regarding the auto expansion?

@aravipra I would also recommend add some note [here](https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/4.14/html-single/storage/index#microshift-lvm-thin-volumes_volume-snapshots-microshift) and point this [RHEL doc](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/configuring_and_managing_logical_volumes/index#creating-thinly-provisioned-logical-volumes_creating-and-managing-thin-provisioned-volumes) 


The problem here is that we may need to enhance a bit RHEL docs to talk about `thin_pool_autoextend_threshold` and `thin_pool_autoextend_percent` parameters configured with /etc/lvm config files. 

I'm maybe wrong here, but the only place that've found some good explanation in our end was with `man 7 lvmthin` and searching `/thin_pool_autoextend` 

I'll raise another issue to RHEL docs to double check this one.

Reply:
I could move the position of the same admonition from line 53 to line 19 as per your suggestion.
The link to the RHEL doc that you have suggested in mentioned in the additional resources [Creating thinly provisioned logical volumes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/creating-and-managing-thin-provisioned-volumes_configuring-and-managing-logical-volumes#creating-thinly-provisioned-logical-volumes_creating-and-managing-thin-provisioned-volumes)

Reply:
Ok so I have moved the admonition position as discussed and the links are added.